src/workers/services/workerCryptoService.ts

```diff
@@ -0,0 +1,57 @@
+import { EncryptionType } from '../../enums';
+import { CipherString, SymmetricCryptoKey } from '../../models/domain';
+import { NodeCryptoFunctionService } from '../../services/nodeCryptoFunction.service';
+import { WorkerLogService } from './workerLogService';
+
+export class WorkerCryptoService {
+    cryptoFunctionService = new NodeCryptoFunctionService();
+    worker: Worker;
+    logService: WorkerLogService;
+    key: any;
```

Comment:
I worry about cleaning up state in this web worker. We would need to make sure locked app states are not holding onto sensitive data.

Reply:
Noted, I'll look into how to handle this